### DISCUSSION TITLE
Connect To External Database with AIO Nextcloud

### DISCUSSION THREAD

**Discussion by chukun09** [nextcloud/all-in-one#4385] [Questions] [answered]
Hi guy,
I'm deploying AIO Nextcloud but i have some issues. I want to deploy Nextcloud in two servers and use the same data folder and database. With data folder, i can use NFS to solve this problem but i don't know how can use external database (Postgres) instead of database of AIO (nextcloud-aio-database) ? Please, help me.

**Comment by szaimen** [comment, marked as answer]
Hi, doing this is actually disrecommended so I will not help you on this. See https://github.com/nextcloud/all-in-one/blob/main/multiple-instances.md if you need multiple indtances.
